commander/test/bootstrapping/commands/generator.spec.ts

```diff
@@ -108,19 +120,53 @@ describe('forging', () => {
 			});
 		});
 
+		describe('when use-status-value is used', () => {
```

Comment:
It would be nice to add a unit test which will assert that enable command correctly populates all 3 height attributes using status values, when `use-status-value` and 1 or more `*height*` flags are used together.

Or otherwise the command should throw an error if it's not allowed to combine those flags.